drivers/storage/rbd/storage/rbd_storage.go

```diff
@@ -47,7 +45,7 @@ func (d *driver) Name() string {
 // Init initializes the driver.
 func (d *driver) Init(context types.Context, config gofig.Config) error {
```

Comment:
Hi @codenrhoden,

Since this PR needs to be rebased onto `master` anyway, can you please fix the name of the `context` variable to be `ctx`. It's the standard way of naming the first parameter of a function when it's of type `Context`. Thank you.